library/formatting.inc.php

```diff
@@ -216,3 +216,24 @@ function oeFormatAge($dobYMD, $nowYMD = '', $format = 0)
 
     return $age;
 }
+
+
+/**
+ * Returns the complete formatted stae string according the global date format
+ * @param $timestamp
+ * @return bool|string
+ */
+function oeFormatLongDate($timestamp)
+{
+    if(!$timestamp){
+        $timestamp = date('Y-m-d H:i');
+    }
+    // assume input is yyyy-mm-dd
+    if ($GLOBALS['date_display_format'] == 1) {      // mm/dd/yyyy, note year is added below
+        $newDate = date('m-d-Y H:i',$timestamp);
```

Comment:
Unless you are using dob's etc :)

Makes more sense now that I see you are using a timestamp, but I don't think this function won't work well with following issues I think:
`$timestamp = date('Y-m-d H:i');` is setting the timestamp to a formatted date
the comment `// assume input is yyyy-mm-dd` doesn't seem right
`$newDate` won't be anything if using yyyy-mm-dd format.

Can you use this anywhere in the code, so can then test it a bit?

And would make function name: 
oeTimestampFormatDateTime()